plugins/switch-config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: switch-config
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/radraw/kube-switch-config
+  shortDescription: "Switches between Kubernetes configuration files"
+  description: |
+    Simple switching between configuration files using symbolic links.
```

Comment:
the terms `configuration files` and `Kubernetes configuration files` don't mean much to kubernetes users (feel free to google them, you won't see them being referred to the same thing).

I'd encourage you to use `kubeconfig file` as the term here for consistency with the rest of the ecosystem.